content/home/examples/a-component-using-external-plugins.js

```diff
@@ -17,16 +17,16 @@ class MarkdownEditor extends React.Component {
   render() {
     return (
       <div className="MarkdownEditor">
-        <h3>Input</h3>
+        <h3>Ulaz</h3>
         <label htmlFor="markdown-content">
-          Enter some markdown
+          Unesite markdown
```

Comment:
Should be `Unesite neku oznaku` or `Unesite neki text`

Reply:
Fixed :+1: